src/MongoDB.Bson/Serialization/BsonClassMap.cs

```diff
@@ -20,7 +20,9 @@
 using System.Linq.Expressions;
 using System.Reflection;
 using System.Runtime.CompilerServices;
+#if NET452 || NETSTANDARD2_0
```

Comment:
without this `#if`, this line is marked as not used by vs or resharper helpers, which confuses a bit